doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -6203,41 +6199,96 @@ \subsubsection{Outputs}\label{outputs-15-002}
-    \item
-    HVAC,Sum,District Heating Hot Water Energy {[}J{]}
+    HVAC,Average,District Heating Water Rate {[}W{]}
     \item
-    Zone,Meter,DistrictHeating:Plant {[}J{]}
+    HVAC,Sum,District Heating Water Energy {[}J{]}
     \item
-    Zone,Meter,Heating:DistrictHeating {[}J{]}
+    Zone,Meter,DistrictHeatingWater:Plant {[}J{]}
     \item
-    HVAC,Average,District Heating Rate {[}W{]}
+    Zone,Meter,Heating:DistrictHeatingWater {[}J{]}
     \item
-    HVAC,Average,District Heating Inlet Temperature {[}C{]}
+    HVAC,Average,District Heating Water Inlet Temperature {[}C{]}
     \item
-    HVAC,Average,District Heating Outlet Temperature {[}C{]}
+    HVAC,Average,District Heating Water Outlet Temperature {[}C{]}
     \item
-    HVAC,Average,District Heating Mass Flow Rate {[}kg/s{]}
+    HVAC,Average,District Heating Water Mass Flow Rate {[}kg/s{]}
 \end{itemize}
 
-\paragraph{District Heating Hot Water Rate {[}W{]}}\label{district-heating-hot-water-rate-w}
-
-\paragraph{District Heating Hot Water Energy {[}J{]}}\label{district-heating-hot-water-energy-j}
+\paragraph{District Heating Water Rate {[}W{]}}\label{district-heating-hot-water-rate-w}
 
-These outputs are the energy taken from purchased hot water. Consumption is metered on Heating:DistrictHeating, DistrictHeating:Plant, and DistrictHeating:Facility.
+This is the useful rate of heating energy from purchased hot water.
 
-\paragraph{District Heating Rate {[}W{]}}\label{district-heating-rate-w}
+\paragraph{District Heating Water Energy {[}J{]}}\label{district-heating-hot-water-energy-j}
 
-This is the useful rate of heating energy from purchased hot water. For the current algorithm, this is the same as District Heating Hot Water Rate.
+These outputs are the energy taken from purchased hot water. Consumption is metered on Heating:DistrictHeatingWater, DistrictHeatingWater:Plant, and DistrictHeatingWater:Facility.
 
-\paragraph{District Heating Inlet Temperature {[}C{]}}\label{district-heating-inlet-temperature-c}
+\paragraph{District Heating Water Inlet Temperature {[}C{]}}\label{district-heating-inlet-temperature-c}
 
-\paragraph{District Heating Outlet Temperature {[}C{]}}\label{district-heating-outlet-temperature-c}
+\paragraph{District Heating Water Outlet Temperature {[}C{]}}\label{district-heating-outlet-temperature-c}
 
-\paragraph{District Heating Mass Flow Rate {[}kg/s{]}}\label{district-heating-mass-flow-rate-kgs}
+\paragraph{District Heating Water Mass Flow Rate {[}kg/s{]}}\label{district-heating-mass-flow-rate-kgs}
 
 These outputs are the supply-side plant loop hot water inlet and outlet temperatures and mass flow rate.
 
+\subsection{DistrictHeating:Steam}\label{districtheatingsteam}
+
+When the user is not interested in a plant simulation or there is some centralized source of steam (district heating steam), the following object can be used in the input.
+
+\subsubsection{Inputs}\label{inputs-17-006}
+
+\paragraph{Field: Name}\label{field-name-16-006}
+
```

Comment:
There are a couple TeX labels which are repeated, throwing warnings.  These need to be tidied up before merge.  It looks like if you just modify these two labels it will be fixed.  You can also just remove the `label` command entirely if these aren't being referenced anywhere.

Reply:
:+1: